List blocked_queries in Experimental features (#6060)
grafana/mimir: docs/sources/mimir/configure/about-versioning.md

  - Instant query splitting (`-query-frontend.split-instant-queries-by-interval`)
  - Lower TTL for cache entries overlapping the out-of-order samples ingestion window (re-using `-ingester.out-of-order-allowance` from ingesters)
  - Use of Redis cache backend (`-query-frontend.results-cache.backend=redis`)

>   - Queries blocking defined on a per-tenant basis and configured via the limit `blocked_queries`

Reviewer: ```suggestion
  - Query blocking on a per-tenant basis (configured with the limit `blocked_queries`)
```
